test/CSV/unQuoted/nestedJson3.csv

```diff
@@ -0,0 +1,5 @@
+carModel,priceRange.min,priceRange.max,priceRange
+Audi,9000,11000,{"min":"9000","max":"11000"}
+BMW,14000,16000,{"min":"14000","max":"16000"}
+Mercedes,19000,21000,{"min":"19000","max":"21000"}
```

Comment:
Well it only appears if the there are values with different data types for one field. I agree that it's some kind of repetition, but on the other hand the resulting csv can be better analyzed if it contains arrays since every field has its own column.